.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,39 @@
+# Description
+
+Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
+
+Fixes #
+
+## Type of change
+
+Please mark relevant options with an `x` in the brackets.
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] This change requires a documentation update
+- [ ] Algorithm update - updates algorithm documentation/questions/answers etc.
+- [ ] Other (please describe):
+
+# How Has This Been Tested?
+
+Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration
+
+- [ ] Integration tests
+- [ ] Unit tests
+- [ ] Manual tests 
+- [ ] No tests required
+
+# Reviewer checklist
+
+Mark everything that needs to be checked before merging the PR.
+
+- [ ] Check if the code is well documented
+- [ ] Check if the behavior is what is expected
```

Comment:
I think this one is ok if you can check it without having to pull the branch and test. That shouldn't be required of the reviewer don't you think?

I still think that it's good to check in PR through screenshots and description though.

Reply:
Yeah. Let's do this and see how we like it for the next few days and promotr